CMakeLists.txt

```diff
@@ -226,6 +226,10 @@ if (UNIX)
     else()
         option(ISPC_STATIC_LINK "Link statically (all except glibc)" OFF)
     endif()
+    if (ISPC_STATIC_LINK)
+        set(ISPC_STATIC_STDCXX_LINK ON)
```

Comment:
I'm confused with the naming. `ISPC_STATIC_STDCXX_LINK` refers to `libstdc++`, while `ISPC_USE_LLVM_LIBCXX` refers specifically to `libc++`. As I understand they can be used together. That's confusing.

Reply:
I understand that in the other way. I treated `ISPC_STATIC_STDCXX_LINK` as link statically any of `libstdc++`, `libc++`. Whereas, `ISPC_USE_LLVM_LIBCXX` is just link with `libc++` dynamically or statically.

But, the other problem that the difference between `ISPC_STATIC_LINK` and `ISPC_STATIC_STDCXX_LINK` will be very subtle after we merge #2895.

Reply:
I have rebased on the current main and removed `ISPC_STATIC_STDCXX_LINK`.